### PR TITLE
Feat/add easy zeit deployment

### PR DESCRIPTION
**Pull request by insanoid**

### CODE REVIEW

File: now.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "version": 2,
+  "builds": [{ "src": "index.js", "use": "@now/node@latest" }],
+  "routes": [
+    {
+      "src": "/stops/nearby",
+      "dest": "/index.js",
+      "methods": ["GET"]
```
Comment (derhuerst) [inline now.json]: Is it necessary to explicitly allow `OPTIONS` here for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)?

**Review by derhuerst — CHANGES_REQUESTED** [review]
Your commit looks fine, but if you PR against the `2` branch, you need to branch off `2`. You can do it e.g. like this:

```shell
git remote add derhuerst https://github.com/derhuerst/bvg-rest.git
git fetch derhuerst
git checkout feat/add-easy-zeit-deployment
git reset --hard derhuerst/2
git cherry-pick aa48f04
git push origin feat/add-easy-zeit-deployment --force
```

**Comment by derhuerst** [comment]
Please also add a `/docs` route.

File: now.json (hunk 1 above)
Comment (insanoid) [reply now.json]: I am not sure what you mean, I can remove `methods` to allow all of the methods (including `OPTIONS`. Is that what you want?

Comment (derhuerst) [reply now.json]: Is `OPTIONS` implicitly/automatically allowed with now.sh? Or do we need to add it here explicitly? You can check with `curl -X OPTIONS …`.

**Comment by derhuerst** [comment]
Thanks! 🎉

**Comment by derhuerst** [comment]
[Now.sh support for regular Express-based APIs seems to be deprecated.](https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/legacy-serverful-behavior)

I could design [`hafas-rest-api`](https://www.npmjs.com/package/hafas-rest-api) to allow for [the `api/*.js` structure encouraged by Now](https://vercel.com/docs/runtimes#official-runtimes/node-js), but the "pass-a-[`hafas-client`](https://www.npmjs.com/package/hafas-client)-instance-in" nature of `hafas-rest-api` would make this quite difficult. This is why I will **remove out-of-the-box Now support in the next major version of `bvg-rest`**.

**Pull request merged**